src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private final String separatorItem = ",";
+    private final String firstItem = "supply";
+    private final String secondItem = "buy";
+    private int totalSupply = 0;
+    private int totalBuy = 0;
+    private int result = 0;
+    private String[] dataArray;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFileMethod(fromFileName);
+        writeDataToFileMethod(toFileName);
```

Comment:
lets refactor a little bit logic - lets divide logic
```suggestion
        String data = readFromFile(fromFileName);
        String report = getReport(data);
        writeToFile(report, toFileName);
```